lib/object/trailer.js

```diff
@@ -14,10 +22,21 @@ class PDFTrailer extends PDFDictionary {
     const id = (new PDFString(info.id)).toHexString()
     this.set('ID', new PDFArray([id, id]))
 
-    this.set('Info', new PDFDictionary({
+    const infoDictionary = {
       Producer: new PDFString(info.producer),
       CreationDate: new PDFString(formatDate(info.creationDate || new Date)),
-    }))
```

Comment:
Just a small nit, do you mind removing the `Producer` so that this value is added as all other properties below?